<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>848</width>
    <height>487</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Demo ColibriNANO</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QLabel" name="lbDevices">
      <property name="text">
       <string>Devices:</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QComboBox" name="cbDevices">
      <property name="minimumSize">
       <size>
        <width>100</width>
        <height>0</height>
       </size>
      </property>
     </widget>
    </item>
    <item row="0" column="2">
     <widget class="QPushButton" name="pbOpen">
      <property name="text">
       <string>Open</string>
      </property>
      <property name="checkable">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item row="0" column="3">
     <spacer name="horizontalSpacer">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>40</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item row="1" column="0" colspan="4">
     <widget class="QWidget" name="wgtReceiver" native="true">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout">
       <property name="spacing">
        <number>0</number>
       </property>
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <widget class="QLabel" name="label">
           <property name="text">
            <string>Frequency:</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QDoubleSpinBox" name="sbFrequency">
           <property name="suffix">
            <string> MHz</string>
           </property>
           <property name="decimals">
            <number>6</number>
           </property>
           <property name="minimum">
            <double>1.000000000000000</double>
           </property>
           <property name="maximum">
            <double>800.000000000000000</double>
           </property>
           <property name="singleStep">
            <double>0.001000000000000</double>
           </property>
           <property name="value">
            <double>7.100000000000000</double>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_2">
           <property name="text">
            <string>Preamp:</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QDoubleSpinBox" name="sbPreamp">
           <property name="suffix">
            <string> dB</string>
           </property>
           <property name="decimals">
            <number>1</number>
           </property>
           <property name="minimum">
            <double>-31.500000000000000</double>
           </property>
           <property name="maximum">
            <double>6.000000000000000</double>
           </property>
           <property name="singleStep">
            <double>0.500000000000000</double>
           </property>
           <property name="value">
            <double>0.000000000000000</double>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="lbSampleRate">
           <property name="text">
            <string>Sample rate:</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QComboBox" name="sbSampleRate">
           <item>
            <property name="text">
             <string>48 kHz</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>96 kHz</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>192 kHz</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>384 kHz</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>768 kHz</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>1536 kHz</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>1920 kHz</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>2560 kHz</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>3072 kHz</string>
            </property>
           </item>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="pbStart">
           <property name="text">
            <string>Start</string>
           </property>
           <property name="checkable">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QRadioButton" name="adcOverload">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <property name="text">
            <string>ADC Overload</string>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer_2">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections>
  <connection>
   <sender>pbOpen</sender>
   <signal>toggled(bool)</signal>
   <receiver>lbDevices</receiver>
   <slot>setDisabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>183</x>
     <y>16</y>
    </hint>
    <hint type="destinationlabel">
     <x>20</x>
     <y>16</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pbOpen</sender>
   <signal>toggled(bool)</signal>
   <receiver>cbDevices</receiver>
   <slot>setDisabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>183</x>
     <y>16</y>
    </hint>
    <hint type="destinationlabel">
     <x>89</x>
     <y>16</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pbStart</sender>
   <signal>toggled(bool)</signal>
   <receiver>lbSampleRate</receiver>
   <slot>setDisabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>388</x>
     <y>43</y>
    </hint>
    <hint type="destinationlabel">
     <x>262</x>
     <y>43</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pbStart</sender>
   <signal>toggled(bool)</signal>
   <receiver>sbSampleRate</receiver>
   <slot>setDisabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>388</x>
     <y>43</y>
    </hint>
    <hint type="destinationlabel">
     <x>317</x>
     <y>43</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pbOpen</sender>
   <signal>toggled(bool)</signal>
   <receiver>wgtReceiver</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>183</x>
     <y>16</y>
    </hint>
    <hint type="destinationlabel">
     <x>423</x>
     <y>245</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
